BootloaderCommonPkg/Library/IppCryptoLib/rsa_verify.c

```diff
@@ -63,37 +69,44 @@ int VerifyRsaSignature (CONST PUB_KEY_HDR *PubKeyHdr, CONST SIGNATURE_HDR *Signa
   bn_rsa_e = (IppsBigNumState *)(bn_buf + sz_adj);
   sz_adj = sz_adj + ((sz_e + sizeof(Ipp32u)) & ~(sizeof(Ipp32u)-1));
 
-  if (sz_adj > sizeof(bn_buf)) {
+  if (sz_adj > bn_buf_sz) {

```

Comment:
Why don't you calculate sz_adj first, then allocate necessary memory?

Reply:
let us try to allocate the required size from beginning.